### DISCUSSION TITLE
Different owner for config file

### DISCUSSION THREAD

**Discussion by bilogic** [coder/code-server#5856] [Q&A] [answered]
1. In Ubuntu, I'm able to start `code-server` for the user dev1 from CLI or `systemd`
2. However, the config file `--config=/path/to/dev1/code-server.yaml` must also be owned by dev1
3. There are certain settings such as `disable-file-downloads` that I wish to enforce
4. I would like `dev1/code-server.yaml` to be owned by the admin and out of reach for dev1, is that possible?

**Comment by jsjoeio** [comment]
great question! I bet @code-asher might know the answer 🤔

**Comment by lg-nineoneninethreenine** [comment]
Why not change the perms with `chown`? I think it would be effective, ie. The user shall have the configuration files in his directory, but the admin only can edit it.

**Comment by code-asher** [comment, marked as answer]
Command-line flags should take precedence over the config so you could enforce them that way.  For example start code-server with `code-server --disable-file-downloads`.
